Data/Sys/GameSettings/SW6.ini

```diff
@@ -0,0 +1,5 @@
+# SW6P78, SW6E78 - WWE'12
+
+[Video_Settings]
+# Fixes Create a Superstar invisible items.
```

Comment:
I would place the "Create a Superstar" phrase between quotes so it's more readable. Ditto on the othher WWE games.

Reply:
That makes sense, thanks.